OWML.Common/Menus/ILayoutManager.cs

```diff
@@ -0,0 +1,18 @@
+﻿using UnityEngine;
+using UnityEngine.UI;
+
+namespace OWML.Common.Menus
+{
+    public interface ILayoutManager
```

Comment:
Might wanna consider changing the name to IModLayoutManager (and similarly for the implementation), just to follow the (stupid) convention of most other files

Reply:
done